src/armado/cdpindex.py

```diff
@@ -132,7 +132,8 @@ def generate_from_html(dirbase, verbose):
         # so we use the words founded in the filename
         # it isn't the optimal solution, but works
         words, title = filename2words(orig)
-        redirs.setdefault(dest, []).append((words, title))
+        # use a set for redirects to avoid duplicated titles after normalization
+        redirs.setdefault(dest, set()).add((tuple(words), title))
```

Comment:
Cuando es un built-in que es sólo sintáxis, como el `[]` no es tan importante, pero cuando ya se llama a una función, está bueno pasar a usar `defaultdict` en vez de `setdefault`, por performance (pensá que con setdefault el tipo está SIEMPRE llamando a set(), incluso cuando luego va y "dest" ya estaba en el dict)

Reply:
Muy buena la observación, cambiado en 06a915e